blog/content/authors/pawel-zaborny/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Paweł Zaborny"
+identifier: pawel.zaborny
+photo:
+facebook:
+twitter:
+linkedin:
+github: "https://github.com/pawelzaborny"
+contribution:
+---
+
+paste the bio here
```

Comment:
@pawelzaborny please describe your biography here - you can check other profiles for examples. You can also add social-media links if you want in the header file - I have added your github profile.